plans/dockerregistry.tf

```diff
@@ -0,0 +1,33 @@
+resource "azurerm_resource_group" "dockerregistry" {
+  name     = "${var.prefix}dockerregistry"
+  location = "${var.dockerregistrylocation}"
+  tags {
+    "env" = "${var.prefix}"
+  }
+}
+
+resource "azurerm_storage_account" "dockerregistry" {
+    name                = "${var.prefix}dockerregistry"
+    resource_group_name = "${azurerm_resource_group.dockerregistry.name}"
+    location            = "${var.dockerregistrylocation}"
+    depends_on          = ["azurerm_resource_group.dockerregistry"]
+    account_type        = "Standard_GRS"
+    tags {
+        "env" = "${var.prefix}"
+    }
+}
+
+resource "azurerm_template_deployment" "dockerregistry"{
+  name  = "${var.prefix}dockerregistry"
+  resource_group_name = "${ azurerm_resource_group.dockerregistry.name }"
+  depends_on          = ["azurerm_resource_group.dockerregistry"]
+  parameters = {
+	registryName = "${var.prefix}registry"
+ 	registryLocation = "${var.dockerregistrylocation}"
+	registryApiVersion = "2016-06-27-preview"
+	storageAccountName = "${ azurerm_storage_account.dockerregistry.name }"
+	#adminUserEnabled = true	
```

Comment:
Does this imply that the we will be unable to administer the configured registry?